docs/markdown/Using Pants/concepts/goals.md

```diff
@@ -60,13 +59,12 @@ You can use several argument types:
 | File path                       | Match the file                              | `pants test project/tests.py` |
 | Directory path                  | Match everything in the directory           | `pants test project/utils`    |
 | `::` globs                      | Match everything in the directory and below | `pants test project::`        |
+| `:` globs                       | Match everything in the current directory   | `pants test :`                |
```

Comment:
I think you might use `project:` to watch the recursive glob example

Reply:
Thanks for taking a look,  Josh. Sorry, I don't follow, what do you mean?

Reply:
The line above this uses `panta test project::`

This line ought to use `pants test project:` for symmetry

Reply:
Oh I see what you mean now. I've decided to skip this example instead and focus on using plain `::` and `:` in the text after the table instead. I think this is easier to illustrate with an example.

Reply:
Oh I think that entry was good, it just needed tweaking

Reply:
Nah, `pants test project:` and `pants test project` are identical in terms of what targets are discovered, so that feels redundant now. I was only interested in `::` and `:` to address items in the root directory - lots of people ask me how to get the root directory items only (they have lots of targets in the root BUILD file) that's why I wanted to add that sentence.